pulp_ansible/app/viewsets.py

```diff
@@ -234,19 +332,88 @@ class CollectionDeprecatedViewSet(ContentViewSet):
     endpoint_name = "collection_deprecations"
     queryset = AnsibleCollectionDeprecated.objects.all()
     serializer_class = CollectionSerializer
+    queryset_filtering_required_permission = "ansible.view_collection"
+
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list", "retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+            },
+            {
+                "action": "update",
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": [
+                    "has_model_or_obj_perms:ansible.change_collection",
+                    "has_model_or_obj_perms:ansible.view_collection",
+                    "has_model_or_obj_perms:ansible.change_ansiblerepository",
+                ],
+            },
+        ],
+        "queryset_scoping": {"function": "scope_queryset"},
+    }
 
 
-class RoleRemoteViewSet(RemoteViewSet):
+class RoleRemoteViewSet(RemoteViewSet, RolesMixin):
     """
     ViewSet for Role Remotes.
     """
 
     endpoint_name = "role"
     queryset = RoleRemote.objects.all()
     serializer_class = RoleRemoteSerializer
-
-
-class GitRemoteViewSet(RemoteViewSet):
+    queryset_filtering_required_permission = "ansible.view_roleremote"
+
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
```

Comment:
You added the `RolesMixin`, but forgot to add the actions from it to the policy.